.github/workflows/cml-with-dvc.yaml

```
@@ -19,25 +18,31 @@ jobs:
         run: |
           cml runner \
               --cloud=aws \
-              --labels=cml-runner \
               --cloud-region=eu-west \
               --cloud-type=t2.micro \
```

`t2.micro` isn't particularly great compared to native GHA. Maybe use [something more beefy](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#machine-type)?

```suggestion
              --cloud-type=m \
```

---

Hmm, not sure. Training the model isn't actually the interesting part of the guide. And `t2.micro` is included in the free tier.